bindings/rust/wasmedge-sys/src/executor.rs

```diff
@@ -6,10 +8,22 @@ use crate::{
 };
 use std::ptr;
 
+/// Struct of WasmEdge Executor.
 pub struct Executor {
     ctx: *mut wasmedge::WasmEdge_ExecutorContext,
 }
 impl Executor {
+    /// Creates a new [`Executor`] to be associated with the given [`Config`] and [`Statistics`].
+    ///
+    /// # Arguments
+    ///
+    /// - `config` specifies the configuration of the new [`Executor`].
+    ///
+    /// - `stat` specifies the [`Statistics`] needed by the new [`Executor`].
```

Comment:
`Statistics` is optional. If users do not want to enable `Statistics` module, they can ignore this argument.

Reply:
Accepted. We'll add the info. Thanks a lot!